src/Bicep.Core.UnitTests/Extensions/TestCompilerExtensions.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Bicep.Core.Features;
+using Bicep.Core.UnitTests.Features;
+using Bicep.TextFixtures.Utils;
+
+namespace Bicep.Core.UnitTests.Extensions
+{
+    public static class TestCompilerExtensions
```

Comment:
Can we move the class to the Bicep.TestFixtures project? The goal is to keep Bicep.Core.UnitTests focused solely on tests, and avoid mixing in shared test utilities used by other test projects. I plan to move the existing utilities incrementally, but for new code, we can start placing it directly in the new project.

Reply:
Added a method but for now, it has type params for the unmigrated types. The types depend on various Bicep.Core.UnitTests constants and a type or 2 so deferring migrating those for now. I left a note on the method.

Reply:
Makes sense.